setup.py

```diff
@@ -50,7 +50,7 @@ def main():
     version = sys.version_info[:2]
     virtualenv_open = ['virtualenv>=1.11.2']
     virtualenv_capped = ['virtualenv>=1.11.2,<14']
-    install_requires = ['py>=1.4.17', 'pluggy>=0.3.0,<1.0', 'six']
+    install_requires = ['setuptools!=34,!=35', 'py>=1.4.17', 'pluggy>=0.3.0,<1.0', 'six']
```

Comment:
This will add a dependency `tox->setuptools`, there isn't really a good way to indicate a "conflicts" in setup.py

tox doesn't really have a runtime need for setuptools so I don't think this is appropriate for `setup_requires`

Reply:
I agree, this was simply one option presented. 

Another option would be to have a note in the docs, and to fix the issue in tox's own CI. Instead of `pip install -U --pre tox`, do `pip install -U --pre setuptools tox`.